core/src/sumeragi/main_loop.rs

```diff
@@ -181,7 +177,7 @@ impl<F: FaultInjection> SumeragiWithFault<F> {
             data: NetworkMessage::SumeragiPacket(Box::new(packet.into())),
             peer: peer.clone(),
         };
-        self.broker.issue_send_sync(&post);
+        self.network.do_send_sync(post);
```

Comment:
IMO it's not good to have a public method prefixed `do_`

Reply:
I don't think it really matters.

Reply:
After we fully move to tokio actors we will deal with actor handle methods like this `network.post(msg)`.
So it's temporary state with `do_send...`.